docker/templates/Dockerfile.tests.m4

```diff
@@ -17,7 +17,7 @@ run_with_apt_cache \
     --mount=type=bind,source=docker/nodesource_pubkey.txt,target=/etc/apt/keyrings/nodesource.asc \
     --mount=type=bind,source=docker/pgdg_pubkey.txt,target=/etc/apt/keyrings/pgdg.asc \
     keep_apt_cache && \
-    apt_install(``ca-certificates curl gnupg'') && \
+    apt_install(``ca-certificates curl gnupg software-properties-common'') && \
```

Comment:
I moved this from `selenium_caa_deps` because it provides the add-apt-repository command used three lines below. I guess it's already being installed by other means, anyway, since that command hasn't failed.

Reply:
Thanks for explaining. Please make this change a separate commit too.